python/openai/README.md

```diff
@@ -171,20 +251,45 @@ pip install -r requirements.txt
 2. Launch the OpenAI server:
 ```bash
 # NOTE: Adjust the --tokenizer based on the model being used
-python3 openai_frontend/main.py --model-repository tests/tensorrtllm_models --tokenizer meta-llama/Meta-Llama-3.1-8B-Instruct
+python3 openai_frontend/main.py --model-repository path/to/models --tokenizer meta-llama/Meta-Llama-3.1-8B-Instruct
 ```
 
 3. Send a `/v1/chat/completions` request:
   - Note the use of `jq` is optional, but provides a nicely formatted output for JSON responses.
 ```bash
+# "ensemble" can also be used here instead of "tensorrt_llm_bls" if setup
```

Comment:
I think the fact that Triton CLI will generate a model whose name is neither `tensorrt_llm_bls` or `ensemble` will be confusing. So I tried to stress the logic behind the variable name choice here. Let me know what you think.
```suggestion
# MODEL should be the client-facing model name in your model repository for a pipeline like TRT-LLM.
# For example, this could also be "ensemble", or something like "gpt2" if generated from Triton CLI
```

Reply:
PR LGTM otherwise

Reply:
I this change is good because it provides more context and reduces the chance of confusion for someone going through the motions.